tm2/pkg/sdk/bank/keeper.go

```diff
@@ -128,15 +130,18 @@ func (bank BankKeeper) SubtractCoins(ctx sdk.Context, addr crypto.Address, amt s
 	}
```

Comment:
if acc not exist, we can halt here, right?

Reply:
that's right, this's an exception that shouldn't happen

Reply:
https://github.com/gnolang/gno/pull/2671/commits/33b5d169d182e9fb2c9bc22ced87f1d7d4ac017d